### PR TITLE
clean up the docs of some modules under lib/js

### PR DESCRIPTION
**Pull request by ringabout**
- clean up the docs
- fixees https://github.com/timotheecour/Nim/issues/64#issuecomment-753722474

### CODE REVIEW

File: lib/js/asyncjs.nim [hunk 1]
```diff
@@ -57,10 +57,10 @@
 ## If you need to use this module with older versions of JavaScript, you can
 ## use a tool that backports the resulting JavaScript code, as babel.
 
-import jsffi
-import macros
+import std/jsffi
+import std/macros
 
-when not defined(js) and not defined(nimdoc) and not defined(nimsuggest):
```
Comment (ringabout) [inline lib/js/asyncjs.nim:63]: see https://github.com/nim-lang/fusion/pull/57#discussion_r551087943

**Review by timotheecour — DISMISSED** [review]
LGTM; in future work
```nim
when not defined(js):
  {.error: "This module only works on the JavaScript platform".}
```
would need to use fatal instead, see rationale here: https://github.com/timotheecour/Nim/issues/64#issuecomment-753905031

EDIT: some tests fail that are related to this PR ...

**Comment by timotheecour** [comment]
> not correct now

proposed fix: simply remove `lib/js/jscore.nim` from `withoutIndex` in kochdocs.nim
=> this will then correctly run docs for `lib/js/jscore.nim`

(another fix would've been this:
```
   for d in items(withoutIndex):
-    commands[i] = nim & " doc $# --git.url:$# -o:$# $#" %
-      [nimArgs, gitUrl,
+    let extra = if isJsOnly(d): "--backend:js" else: ""
+    commands[i] = nim & " doc $# $# --git.url:$# -o:$# $#" %
+      [extra, nimArgs, gitUrl,
       destPath / changeFileExt(splitFile(d).name, "html"), d]
     i.inc

```
but the 1st proposed fix is better/simpler)

**Comment by ringabout** [comment]
I see, thanks, you are always very helpful!(I was testing it too)

**Review by timotheecour — APPROVED** [review]
LGTM again (if passes CI)

**Pull request merged**